internals/core_models.py

```diff
@@ -954,3 +954,206 @@ def put(self, notify=True, **kwargs):
   anticipated_spec_changes = ndb.StringProperty()
 
   finch_url = ndb.StringProperty()
+
+
+# Note: This class is not used yet.
+class FeatureEntry(ndb.Model):  # Copy from Feature
+  """This is the main representation of a feature that we are tracking."""
+
+  # Metadata: Creation and updates.
+  created = ndb.DateTimeProperty(auto_now_add=True)
+  updated = ndb.DateTimeProperty()
+  accurate_as_of = ndb.DateTimeProperty()
+  creator = ndb.StringProperty()
+  updater = ndb.StringProperty()
+
+  # Metadata: Access controls
+  owners = ndb.StringProperty(repeated=True)  # copy from owner
+  editors = ndb.StringProperty(repeated=True)
+  unlisted = ndb.BooleanProperty(default=False)
+  deleted = ndb.BooleanProperty(default=False)
+
+  # Descriptive info.
+  name = ndb.StringProperty(required=True)
+  summary = ndb.TextProperty(required=True)
+  category = ndb.IntegerProperty(required=True)
+  blink_components = ndb.StringProperty(repeated=True)
+  star_count = ndb.IntegerProperty(default=0)
+  search_tags = ndb.StringProperty(repeated=True)
+  feature_notes = ndb.TextProperty()  # copy from comments
+
+  # Metadata: Process information
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  bug_url = ndb.StringProperty()
```

Comment:
Should this be `tracking_bug_url`? That's what it's called in the interface.

Reply:
I was thinking the same thing.   I'll make that change in a follow-up PR.

Reply:
I started making that follow-up PR and I realized that a fair number of details in the code need to change for every field that we rename.  So, even though clarifying the source code is one of the goals of this effort, I think we need to be selective about how many things we rename.  bug_url is not as good as tracking_bug_url, but I think it can be clarified in the help text and a comment on the field name rather than changing this particular field name.

Reply:
That's fair.